install/gcp/installer/util/gcloud.py

```diff
@@ -51,6 +51,24 @@ def get_gcloud_info():
             sys.exit(1)
     return project_id, authed_user, is_devshell
 
+def activate_service_account(service_account, key_path):
+    """Activate the service account with gcloud.
+
+    Args:
+        service_account (str): Service account email
+        key_path (str): Absolute path to service account key file
+    """
+
+    return_code, _, err = utils.run_command(
+        ['gcloud', 'auth', 'activate-service-account',
+         service_account, '--key-file=' + key_path])
```

Comment:
Now that I'm looking at this `key_path` naming with the gcloud command's naming, should would align them?
i.e. change `key_path` to `key_file` everywhere, including the flag?  How do you think?
```
'--key-file=' + key_file
```

Reply:
Done